src/Apps/Auction/Components/BidForm.tsx

```diff
@@ -111,13 +113,15 @@ export const determineDisplayRequirements = (
 }
 
 export const BidForm: React.FC<Props> = ({
+  artworkSlug,
   initialSelectedBid,
   me,
   onSubmit,
+  relay,
   saleArtwork,
-  showPricingTransparency = false,
   trackSubmissionErrors,
 }) => {
+  const { ENABLE_PRICE_TRANSPARENCY } = sd as any
```

Comment:
You should be able to use the `GlobalData` type, defined here once you add the `ENABLE_PRICE_TRANSPARENCY` mapping: https://github.com/artsy/reaction/blob/master/typings/sharify.d.ts